project_time_range/README.md

```diff
@@ -1,12 +1,18 @@
-# Project Time Range
+Project Time Range
+==================
+- This module adds the fields `Min` and `Ideal` and `Max` under the `Management` tab and `Budget` group of a project. The three fields are computed as the sum of the same fields on the Task Templates linked to the Project
 
-This module adds the fields `Min` and `Ideal` and `Max` under the `Management` tab of a project.
-These fields contain the minimum and maximum time initialy estimated on the task.
-
-
-![Task Form](static/description/form_view_project.png?raw=true)
+![](static/description/project_time_range.png?raw=true)
 
+Configuration
+-------------
+No configuration required apart from module installation.
 
 Contributors
 ------------
 * Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+* Komit (https://komit-consulting.com)
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6260167)